src/site/stages/build/process-cms-exports/transformers/node-press_releases_listing.js

```diff
@@ -4,6 +4,23 @@ const {
   utcToEpochTime,
 } = require('./helpers');
 
+const reverseFields = reverseFieldList => ({
+  entities: reverseFieldList
+    .filter(
+      reverseField =>
+        reverseField.entityBundle === 'press_release' && reverseField.status,
```

Comment:
Chris recommended we use `reverseField.isPublished` rather than `reverseField.status` for filtering reverse fields. I still need to update this in one of my PR's myself. https://github.com/department-of-veterans-affairs/vets-website/pull/13772/files#r467229463

Reply:
ok, I'll do that